lib/pymedphys/docs/contrib/info/release-guide.md

```diff
@@ -1,7 +1,7 @@
 # pymedphys Release Procedure
 
-.. note::
-Please ensure that you have followed the [setup guide](https://docs.pymedphys.com/en/latest/contrib/setups/index.html)
+
+> **NOTE:** Please ensure that you have followed the [setup guide](https://docs.pymedphys.com/en/latest/contrib/setups/index.html)
```

Comment:
If you want you can change this to:

```{note}
Here is a note
```

If you want to use the note directive within markdown. See https://jupyterbook.org/en/stable/content/myst.html#directives

Reply:
Thanks!

Would you mind if I made the change after the release? The CI is almost done... haha

Reply:
Of course, you can also feel free to not change it :)

Reply:
Also, you don't need to wait for the LGTM analysis